### PR TITLE
🏗 Fix renovate config file

### PR DESCRIPTION
**Pull request by rcebulko**
Fixes errors introduced in previous config changes. Also enables creation of a [master issue](https://docs.renovatebot.com/configuration-options/#masterissue) which will list all planned and pending renovate PRs, useful for tracking and debugging.

Related to https://github.com/ampproject/amphtml/issues/28473

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "promise-pjs": "1.1.4",
     "prop-types": "15.7.2",
     "react-dates": "15.5.3",
+    "renovate": "^19.239.8",
```
Comment (rsimha) [inline package.json:28]: Use exact version here (remove caret).

Comment (rcebulko) [reply package.json]: Removed it entirely; will add as devDependency in the PR introducing the presubmit check

**Comment by rcebulko** [comment]
Update: Switched this to run "at any time" AKA continuously, but it will now wait to create relevant PRs until they are marked on the master issue.

**Comment by rcebulko** [comment]
Related to #28322

**Pull request merged**